locales/en.yml

```diff
@@ -32,24 +32,27 @@ chat:
   title: Chat
   welcome: Start conversation with something nice.
   with: Chat with {username}
-  dateDiff: "{diff} ago"
+  dateDiff: '{diff} ago'
```

Comment:
where is `{diff}`  from?
and that value is always changed or fixed?

Reply:
{diff} it was from an old version of the main code, I have refactor the code now

Reply:
@best-lucky1030 read about i18n formatting here - https://kazupon.github.io/vue-i18n/guide/formatting.html
That's how you can insert dynamic variables into translation. We talk more about it in our daily meetings.